provider/tencentcloud/cloudapi/tencentapi.go

```diff
@@ -18,6 +18,7 @@ package cloudapi
 
 import (
 	"encoding/json"
+	errs "errors"
```

Comment:
Instead of setting an alias on std errors, wdyt of setting an alias on the specific tencentcloud errors pkg ? (like for dnspod and privatedns)

Reply:
make sense

Reply:
changed